### PR TITLE
Adding new Reference video section in the MB page

### PR DESCRIPTION
**Pull request by omsuneri**
**Adding section along with playable youtube videos of Music Blocks
**also improving some paragragh alingment in MB page

### CODE REVIEW

**Comment by omsuneri** [comment]
@pikurasa it would be nice if you merge this PR under hacktoberfest tag 😁

File: music-blocks.html [hunk 1]
```diff
@@ -144,6 +144,34 @@ <h2>CREDITS</h2>
...
 
+<!-- "Refence Video" section -->
+<section id="testimonial" >
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12" >
+                <div class="row justify-content-center">
+                    <div class="col-md-8 col-md-push-2">
+                        <h2 class="text-center">REFERENCE VIDEOS</h2>
+                        <p class="customParagraphStyle">Explore more about Sugar labs and Music Blocks on our official youtube channel<a href="https://www.youtube.com/@SugarlabsOrg-EN" class="hrefCustomColor"> Sugar Labs</a>.</p>
```
Comment (pikurasa) [inline music-blocks.html:155]: I think we should link to our Music Blocks playlist instead: https://www.youtube.com/watch?v=v8-M81pp5ng&list=PLyTz5XRZyi-xRj6aca64IrXtjLeMydWrW&pp=gAQBiAQB

File: music-blocks.html [hunk 2]
```diff
@@ -144,6 +144,34 @@ <h2>CREDITS</h2>
...
+                                <iframe width="480" height="280"
+                                src="https://www.youtube.com/embed/actLKGJVa_Q">
+                                </iframe> 
+                            </div> 
+                            <div>  
+                                <iframe width="480" height="280"
+                                src="https://www.youtube.com/embed/RngzuKVl2Xk"> 
+                                </iframe>
+                            </div> 
+                        </div>
```
Comment (pikurasa) [inline music-blocks.html:167]: In addition to the two videos above, please also add:

https://www.youtube.com/watch?v=AZLQCDlKqdc

**Review by pikurasa — CHANGES_REQUESTED** [review]

File: music-blocks.html (hunk 1 above)
Comment (omsuneri) [reply music-blocks.html:155]: So should I add more such type of iframe or just add a simple clickable link there below

File: music-blocks.html (hunk 2 above)
Comment (omsuneri) [reply music-blocks.html:167]: So I will make these changes by this Sunday as I ll be traveling to my home for the next 2 days so will update you regarding this changes on Sunday

**Pull request closed without merging**